front/src/app/building-objects-if.ts

```diff
@@ -28,15 +28,15 @@ export interface section {
   end: coordinate;
 }
 
-export interface room {
+export class room {
   name: string;
   sections: section[];
   alias: string;
   doors: section[];
   fill: string;
```

Comment:
Im Backend sieht die Struktur wie folgt aus:

```
type Room struct {
	Id          int
	Name        string
	Description string
	Alias       []string
	Doors       []Door
	Color       string
	Sections    []Section
	Floor       int
	PathNode    navigation.PathNode
}
```
Die Benennung im JSON wäre dann gleich wie oben in der Struktur.
Sollen wir uns da evtl. nochmal abstimmen? Würde typescript das unterstützen wenn im JSON mehr Daten sind als du abfragst?

Reply:
Floor vom Serializer ignorieren, PathNode als Array
```
type Room struct {
	Id          int
	Name        string
	Description string
	Alias       []string
	Doors       []Door
	Color       string
	Sections    []Section
	Floor       int    `json:"-"`
	PathNode    []navigation.PathNode
}
```